app/src/main/kotlin/com/intive/picover/main/navigation/view/PicoverNavHost.kt

```diff
@@ -60,7 +61,8 @@ fun PicoverNavHost(
 				route = "partyDetails/{partyId}",
 				arguments = listOf(navArgument("partyId") { type = NavType.StringType }),
 			) {
-				PartyDetailsScreen(hiltViewModel())
+				val partyId = it.arguments!!.getString("partyId")!!
+				Navigator(PartyDetailsScreen(partyId))
```

Comment:
Plan is to migrate each child screen to voyager first and after that migrate whole main screen to TabNavigator from voyager, we'll see how it goes 😄 So temporarily each child is wrapped in it's own Navigator to keep all state restoration and view model disposing working correctly